src/logic/notifications/store/reducer/notifications.ts

```diff
@@ -20,7 +20,7 @@ export default handleActions(
       const { dismissAll, key } = action.payload
 
       if (key) {
-        return state.update(key, (prev) => prev.set('dismissed', true))
+        return state.update(key, (prev) => prev?.set('dismissed', true) ?? prev)
```

Comment:
what was the problem with that? looks weird

Reply:
Sorry missed the description. Did you try to debug why prev was undefined?

Reply:
unfortunately no. And now I'm not being able to reproduce it, despite I've had experienced this same issue in the past.

I'll revert those fixes and will open an issue to tackle them independently.

Reply:
finally, if I close the notification before the tx is resolved, then it _crashes_

![image](https://user-images.githubusercontent.com/3315606/92668110-3e4e3700-f2e4-11ea-831b-3b29395368ac.png)

Reply:
https://github.com/gnosis/safe-react/issues/1345